### DISCUSSION TITLE
Errors while running flet program in Ubuntu.

### DISCUSSION THREAD

**Discussion by kanstat** [flet-dev/flet#875] [Q&A]
#### Tried running simple flet program on Ubuntu, on different machines. Getting same following errors.
```
* (flet:103071): CRITICAL **: 21:21:26.265: Failed to read XDG desktop portal settings: GDBus.Error:org.freedesktop.portal.Error.NotFound: Requested setting not found

** (flet:103071): CRITICAL **: 21:21:26.268: Failed to read XDG desktop portal settings: GDBus.Error:org.freedesktop.portal.Error.NotFound: Requested setting not found

(flet:103071): GLib-GObject-WARNING **: 21:21:29.552: ../../../gobject/gsignal.c:2736: instance '0x21ab290' has no handler with id '171'
```
#### Also, I was not able to resize the window after removing Title Bar and Title Buttons using:
```
page.window_title_bar_hidden = True
page.window_title_bar_buttons_hidden = True
```

**_The programs ran properly without errors in Windows OS. How can that be fixed in case of Linux (Ubuntu) ?_**

**Comment by FeodorFitsner** [comment]
What version of Ubuntu are you running it on? Do those errors prevent the app from running or you see app window opened anyway?

**Reply by kanstat** [reply]
The ubuntu version is v20.04. 
The app window does show up, but when I tried hiding the app window Title Bar and buttons in the flet program, I was Not able to resize the app window size from the edge/border or its corner. Tried this on 4 different machines with Ubuntu, getting same results.
(But dragging the app to the rightmost edge of the screen does change the size to fit the half of the screen, like any other app.)
